packages/blockfrost/src/BlockfrostToCore.ts

```diff
@@ -25,7 +25,7 @@ export const BlockfrostToCore = {
 
   currentWalletProtocolParameters: (
     blockfrost: Responses['epoch_param_content']
-  ): ProtocolParametersRequiredByWallet => ({
+  ): Omit<Cardano.ProtocolParameters, 'costModels'> => ({
```

Comment:
```suggestion
  ): ProtocolParametersRequiredByWallet => ({
```
I think it's better to keep the return type and adjust the response, otherwise we should rename it to `NetworkInfo.protocolParameters` and return `Cardano.ProtocolParameters` (including `costModels`).